updateVersion.sh

```diff
@@ -117,6 +118,7 @@ function merge_files() {
   sed -i "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" $ISTIO_AUTH
   cat $AUTH_SRC/istio-ingress-auth.yaml >> $ISTIO_AUTH
   cat $AUTH_SRC/istio-egress-auth.yaml >> $ISTIO_AUTH
+  cp $ISTIO_AUTH $ISTIO_AUTH_PER_CLUSTER_CA
```

Comment:
The cluster CA is deployed separately. I think it's better we decouple the deployment of CA and other components.